src/error-display/error-display.tsx

```diff
@@ -26,7 +28,11 @@ export const ErrorDisplay = ({
     // =============================================================================
     // CONST, STATE, REF
     // =============================================================================
-    const defaultAssets = ERROR_DISPLAY_DATA.get(type);
+    const theme = useTheme();
+    const defaultAssets = errorDisplayData(
+        type,
+        (theme as unknown as ThemeSpec).resourceScheme
```

Comment:
i think i've already typed  styled-components `DefaultTheme`. You should be able to access the `resourceScheme` without typecasting `theme`

Reply:
got it! will remove the type casting here. thanks!